### ISSUE TITLE
README - Create a List of Skills Added 

### ISSUE THREAD

**Issue by tarunnsingh** [alexa-dev-hub/alexa-skills#18]
![image](https://user-images.githubusercontent.com/31896659/94849993-e4a0de80-0443-11eb-832b-6066c41cfbf7.png)

#### Alexa Skills are added in the **alexa-skills-by-the-team** and **alexa-skills-by-the-community** folders.
✔️  Check [HERE](https://github.com/alexa-dev-hub/alexa-skills/tree/master/skills-by-the-community) and [HERE](https://github.com/alexa-dev-hub/alexa-skills/tree/master/skills-by-the-team).

### Requirement is to enlist all these Skills as a List in the README. 📜 
- The List should mention the **Name of the Creator, the Skill name** - clickable link.
- The List should **update each time,** a new skill's PR is created.

Please let us know about your approach before asking to be assigned.

**Comment by tarunnsingh** [comment]
@hariom1625 want to give it a try?

**Comment by hariom1625** [comment]
Thank you @tarunnsingh for mentioning. As of now, I can't think of any GitHub Action to carry out this task, if I will create any GitHub action or come across any solution I will surely create a PR. So, don't assign it to me now let the issue be opened so as if anyone else knows a better solution they can implement it :-)

**Comment by tarunnsingh** [comment]
Alright thanks!

**Comment by cruzcid** [comment]
Actually I was thinking that implement a GitHub Action(this would be the first implementation of that kind though) may be a good solution.

**Comment by tarunnsingh** [comment]
Hey @cruzcid Do you want to try it out then?

**Comment by cruzcid** [comment]
Yes please. I hope to make progress on it. I would let you know what happens with the development :)

**Comment by tarunnsingh** [comment]
@cruzcid any progress on this?

**Comment by cruzcid** [comment]
Hi @tarunnsingh 
I am creating a GitHub action which:
1.- Triggers itself when a pull request is made to a given branch (Does "main" branch is the specific branch that should trigger the action when a pull request is made ?)
2.- Get the pull request list and info pertinent to each pull request.
3.- Write the required information "Name of the Creator", "Skill name"

I have completed steps 1 & 3. I have tested with pull request made to a personal repository and it triggers the Github action and write the file. 

Next steps are:
1. Get the pull request info and create the requested list. 
2. Integrate changes to your repository.

**Comment by tarunnsingh** [comment]
That sounds great @cruzcid !
Yes it has to be the main branch (master branch). 
Take your time, no issues on that 👍

**Comment by cruzCastroMatchcraft** [comment]
Hi. I have found the way to get the pull request list and write to a README file. 
I will clean the code and make a few test cases more before attempting to integrate changes with your code. 

Thank you for the patience!

**Comment by tarunnsingh** [comment]
@cruzCastroMatchcraft @cruzcid Okay Cool!
